platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/markdown/ui/MarkdownPanel.kt

```diff
@@ -148,9 +148,9 @@ private val JewelTheme.Companion.linkColor: Color
   @Composable
   get() {
     return if (isDark) {
-      colorPalette.blue.lastOrNull() ?: Color.Blue
+      colorPalette.blue.lastOrNull() ?: Color.White
     } else {
-      colorPalette.blue.firstOrNull() ?: Color.Blue
+      colorPalette.blue.firstOrNull() ?: Color.White
```

Comment:
hardcoding it to white will be bad on light themes :)

Reply:
Makes sense 😅

Reply:
ahh that's tricky 😄 I switched to use JewelTheme text color instead